tests/integration/devfile/utils/utils.go

```diff
@@ -177,3 +177,25 @@ func ExecPushWithNewFileAndDir(projectDirPath, cmpName, namespace, newFilePath,
 	args = useProjectIfAvailable(args, namespace)
 	helper.CmdShouldPass("odo", args...)
 }
+
+// ExecWithRestartAttribute executes odo push with a command attribute restart
+func ExecWithRestartAttribute(projectDirPath, cmpName, namespace string) {
```

Comment:
This is a utils function, we shouldn't run the entire test here.

Reply:
Hmm, i am not sure about this, other tests are in the same way.

Reply:
Sorry now I get it. These tests are reused for the docker push verification also.